docs/04_User_Guides/autofix_resources.md

```diff
@@ -0,0 +1,55 @@
+[TOC]
+
+## Description, how to apply autoformatting frontend resources
+### What is this?
+In this guide we provide some simple examples for the usage of autofixing functionality.
+Crafty comes with some default linter configuration including among other 
+
+- eslint (js)
+- post-css (css)
+- prettier (js + css)
+
+
+### When should you use it?
+
+Occasionaly you will see in your building logs several linting errors ( eg. eslint mentioning XXX rule is violated).
+
+Some of them need to be fixed manually but not all.
+Before diving into fixing 100 lines of code try these commands they come in handy.
+
+## Option 1 : Crafty jsLint, CssLint options
+
+`crafty jsLint src/** --fix --preset recommended `
+
+will lint all your files and list the errors
+
+
+`crafty cssLint src/** --fix --preset recommended `
+
+more info for crafty eslint options here:  https://swissquote.github.io/crafty/Packages/crafty-preset-eslint/index.html
```

Comment:
You can make this an automatic link by using a relative path `[crafty-preset-eslint](../05_Packages/05_crafty-preset-eslint/index.md)`

Reply:
Also, you mention "eslint" but this is the command to lint CSS, maybe move that to the previous command

Reply:
corrected :)